### PR TITLE
Add min gas and mint schedule proposals

### PR DESCRIPTION
**Pull request by philipsu522**

### CODE REVIEW

File: pacific-1/proposals/08-11-2023-increase-min-gas.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "title": "Update Global Minimum Prices",
+    "description": "This proposal seeks update the global minimum prices for a gas unit.",
+    "changes": [
+      {
+        "subspace": "params",
```
Comment (udpatil) [inline pacific-1/proposals/08-11-2023-increase-min-gas.json:6]: Is this the correct subspace? When i query against RPC, I get this response:
```
seid q params subspace params FeeParams --node https://sei-rpc.polkachu.com
key: FeeParams
subspace: params
value: ""
```

File: pacific-1/proposals/08-11-2023-mint-schedule.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
+{
+  "title": "Mint Param Change Proposal",
+  "description": "Proposal to add Daily Mint Token Release Schedule",
+  "changes": [
+    {
+      "subspace": "mint",
+      "key": "TokenReleaseSchedule",
+      "value": [
+	    {
+	      "token_release_amount": "300000000",
```
Comment (udpatil) [inline pacific-1/proposals/08-11-2023-mint-schedule.json:10]: is this based on a document or something we can check with for correctness?

File: pacific-1/proposals/08-11-2023-mint-schedule.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	      "start_date": "2023-08-15",
+	      "end_date": "2024-08-14"
```
Comment (udpatil) [inline pacific-1/proposals/08-11-2023-mint-schedule.json:12]: Just to double check, is the "end date" inclusive or exclusive?

File: pacific-1/proposals/08-11-2023-mint-schedule.json (hunk 2 above)
Comment (udpatil) [inline pacific-1/proposals/08-11-2023-mint-schedule.json:10]: Is this denoted in microdenoms? (Usei)

File: pacific-1/proposals/08-11-2023-mint-schedule.json (hunk 3 above)
Comment (philipsu522) [reply pacific-1/proposals/08-11-2023-mint-schedule.json:12]: end date is inclusive

File: pacific-1/proposals/08-11-2023-mint-schedule.json [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+	      "start_date": "2027-08-15",
+	      "end_date": "2028-08-14"
+	    },
+	    {
+	      "token_release_amount": "120000000000000",
+	      "start_date": "2028-08-15",
+	      "end_date": "2029-08-14"
+	    },
+	    {
+	      "token_release_amount": "112500000000000",
```
Comment (udpatil) [inline pacific-1/proposals/08-11-2023-mint-schedule.json:40]: missing one more entry of 112500000000000

Comment (udpatil) [reply pacific-1/proposals/08-11-2023-mint-schedule.json:40]: i believe the last item should be ending in 2033 right? so the last two need to be shifted and this one needs to be inserted

**Review by udpatil — APPROVED** [review]
Lgtm

**Pull request merged**